test/main.spec.ts

```diff
@@ -73,21 +73,48 @@ test('Get legacy Config Tokens', async () => {
     expect(Utils.getConfigTokens()).toStrictEqual(new Set(['DUMMY_CONFIG_TOKEN_1', 'DUMMY_CONFIG_TOKEN_2', 'DUMMY_CONFIG_TOKEN_3']));
 });
 
+test("Collect JFrog Credentials from env vars", async () => {
+   process.env['JF_URL'] = '';
+   let jfrogCredentials: JfrogCredentials = Utils.collectJfrogCredentialsFromEnvVars();
+   expect(jfrogCredentials.jfrogUrl).toEqual(undefined);
```

Comment:
Let's do it in all other places in this test
```suggestion
   expect(jfrogCredentials.jfrogUrl).toBeUndefined();
```